### ISSUE TITLE
Prop warning with `inputRef`

### ISSUE THREAD

**Issue by dbalatero** [sanniassin/react-input-mask#123]
I believe `this.props.inputRef` needs to be removed from the `...props` object before getting passed into `<input>`: https://github.com/sanniassin/react-input-mask/blob/master/src/index.js#L525

Otherwise it raises a development error:

```
warning.js:33 Warning: React does not recognize the `inputRef` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `inputref` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in input (created by InputElement)
    in InputElement (created by Phone)
    in div (created by Phone)
    in div (created by Phone)
    in div (created by Phone)
    in Phone (created by ValidationWrapper)
    in ValidationWrapper (created by Connect(ValidationWrapper))
    in Connect(ValidationWrapper) (created by Unknown)
    in Unknown (created by QuestionPanel)
    in fieldset (created by QuestionPanel)
    in div (created by QuestionPanel)
    in div (created by QuestionPanel)
    in QuestionPanel (created by Connect(QuestionPanel))
    in Connect(QuestionPanel) (created by Panel)
    in Panel (created by TransitionMotion)
    in div (created by TransitionMotion)
    in div (created by TransitionMotion)
    in TransitionMotion (created by PanelAnimator)
    in PanelAnimator (created by Connect(PanelAnimator))
    in Connect(PanelAnimator) (created by ScrollingForm)
    in div (created by ScrollingForm)
    in div (created by ScrollingForm)
    in div (created by ScrollingForm)
    in div (created by ScrollingForm)
    in div (created by ScrollingForm)
    in ScrollingForm (created by Connect(ScrollingForm))
    in Connect(ScrollingForm) (created by App)
    in FragmentComponent (created by withContext(FragmentComponent))
    in withContext(FragmentComponent) (created by WithId)
    in WithId (created by getContext(WithId))
    in getContext(WithId) (created by Connect(getContext(WithId)))
    in Connect(getContext(WithId)) (created by App)
    in div (created by App)
    in App (created by AppRoot)
    in FragmentComponent (created by withContext(FragmentComponent))
    in withContext(FragmentComponent) (created by WithId)
    in WithId (created by getContext(WithId))
    in getContext(WithId) (created by Connect(getContext(WithId)))
    in Connect(getContext(WithId)) (created by AppRoot)
    in Provider (created by AppRoot)
    in AppRoot
    in AppContainer
```

**Comment by dbalatero** [comment]
@sanniassin I can contribute a patch if you like.

**Comment by sanniassin** [comment]
Fixed in 1.2.2. Thanks!
